proposals/lip-0070.md

```diff
@@ -49,6 +49,13 @@ Here we define the constants related to reward sharing. All other PoS module con
 | `COMMISSION_INCREASE_PERIOD` | uint32 | 260000 | Determines how frequently (after how many blocks) a validator can increase their commission. |
 | `MAX_COMMISSION_INCREASE`| uint32 | 500 | Determines the maximum allowed increase on the commission per transaction. |
 
+### Event Names
+
+| Name | Type | Value | Description |
+|------|------|-------|-------------|
+| `EVENT_NAME_COMMISSION_CHANGE` | string | "commissionChange" | Used for events emitted whenever validators change their commission. |
+| `EVENT_NAME_REWARDS_ASSIGNED` | string | "rewardsAssigned" | Used for events emitted whenever shared rewards are assigned to stakers. |
```

Comment:
I'm not familiar if there is any convention here, but the constant name and the value of `EVENT_NAME_REWARDS_ASSIGNED` is in past tense, while `EVENT_NAME_COMMISSION_CHANGE` is in present tense.

To me, it makes more sense that both of these events use past tense, as they trigger when the action has occurred.

No matter of which tense is selected, it would make it more consistent if both events use the same tense.

Reply:
The intention is that "assigned" comes from the verb "assign" in past tense, while "change" is just the noun "change". 

The strings used for names are the ones used in SDK implementation and I guess we would not like to change them.